### DISCUSSION TITLE
RelationReference fields can't be properly updated

### DISCUSSION THREAD

**Discussion by EnekoRuiz** [INSPIRE-MIF/helpdesk-registry#67] [Announcements]
This error happens when updating the information on a RelationReference type field. 
If the user removes the value and selects a new one, the Registry creates a draft with the edited version. This draft has the correct information, but when approving the changes to turn it into Valid state, and publish the item, both the old relation and the new one proposed appear in the administration tool and in the Front end.

### How to reproduce
Here's an example illustrated by screenshots to help me explain the issue.

This item has the Application Schema field, which is of type RelationReference.
![image](https://github.com/INSPIRE-MIF/helpdesk-registry/assets/99260687/afab26f9-9393-41ee-a18c-c00e7fc5811d)

Now the Application Schema field is updated.
![image](https://github.com/INSPIRE-MIF/helpdesk-registry/assets/99260687/ad3dafca-6928-41e7-a80d-49e3c6fe80d7)

After clicking on "Accept clarification" a draft is created.
![image](https://github.com/INSPIRE-MIF/helpdesk-registry/assets/99260687/e2589f71-0e0e-4cea-998d-e3331666dbfc)

Now, after approving the changes instead of replacing the information with the new one, both appear.
![image](https://github.com/INSPIRE-MIF/helpdesk-registry/assets/99260687/b5658589-ac90-4c14-800a-eb6a0d10c6c6)

**Note**: this bug has been reproduced in version v2.3.0 (INSPIRE Registry version)

**Comment by arantzaetxebarria** [comment]
Dear community, 

This issue will be fixed in the Re3gistry helpdesk, to be updated, please follow up on the next issue: https://github.com/ec-jrc/re3gistry/issues/312
